packages/ui-components/src/components/select/select.scss

```
@@ -4,16 +4,20 @@
 	/**
 	 * @prop --select-max-height: Select maximum height.
 	 * @prop --select-background-color: Select background color.
+	 * @prop --select-border-width: Select border width.
+	 * @prop --select-border-radius: Select border radius.
 	 * @prop --select-border-color: Select border color.
 	 */
 
 	--select-max-height: auto;
 	--select-background-color: #{kv-color('neutral-7')};
+	--select-border-width: 1px;
+	--select-border-radius: #{$spacing};
 	--select-border-color: #{kv-color('neutral-6')};
```

Nice ideia 👍 

Just a thought two avoid two CSS custom properties for the "same" property:
```suggestion
	--select-border: 1px solid #{kv-color('neutral-6')};
	--select-border-radius: #{$spacing};
```